src/main/app/src/assets/i18n/en.json

```
@@ -458,6 +458,7 @@
   "indicatie.VERGRENDELD": "is locked",
   "indicatie.VERLENGD": "has been extended",
   "indicatie.VERZONDEN": "has been sent",
+  "indicatie.ONTVANGSTBEVESTIGING_NIET_VERSTUURD": "No confirmation email sent",
```

why add `email` here, but not in the other translation? No need I'd say.
```suggestion
  "indicatie.ONTVANGSTBEVESTIGING_NIET_VERSTUURD": "No confirmation sent",
```
or
```suggestion
  "indicatie.ONTVANGSTBEVESTIGING_NIET_VERSTUURD": "No confirmation receipt sent",
```
or leave it as is 😃